server/src/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -926,7 +926,7 @@ protected void checkIfPodIsDeletable(final long podId) {
                 dbName = "cloud";
             }
 
-            String selectSql = "SELECT * FROM `" + dbName + "`.`" + tableName + "` WHERE " + column + " = ?";
+            String selectSql = "SELECT * FROM `?`.`?` WHERE ? = ?";
 
             if(tableName.equals("vm_instance")) {
                 selectSql += " AND state != '" + VirtualMachine.State.Expunging.toString() + "' AND removed IS NULL";
```

Comment:
wondering why are we even using a prepareStatement here (and doing a fix to satisfy findbugs). Why dont we use createStatement() and execute sql directly?